doc/tutorials/pynest_tutorial/part_4_topologically_structured_networks.rst

```diff
@@ -46,9 +46,12 @@ In the previous sections, we saw that we can create deterministic or
 randomly selected connections between networks using ``Connect()``. Likewise, it is
 also possible to use ``Create()`` and ``Connect()`` to create network
 models that incorporate spatial location and spatial connectivity
-profiles. **NOTE:** Full documentation for usage of the spatial functions
-is present in our :doc:`../../guides/guide_structurally_spatial_networks` [1]_,
-which in the following is referenced as full-source.
+profiles.
+
+.. note::
+   Full documentation for the usage of spatial functions
+   is present in our :doc:`../../guides/guide_spatially_structured_networks` [1]_,
+   which in the following is referenced as full-source.
```

Comment:
I find "is present" a bit cumbersome and "full-source" a little informative name. How about
"For comprehensive documentation of spatial properties and connectivity, see the :doc:`../../guides/guide_spatially_structured_networks` [1]_, which is referenced as *Spatial Network Guide* in the following."

How does the link show in the rendered docs?

Furthermore, the reference [1] seems to point to the PDF version of the manual. But that makes no sense. As long as all the material from the manual is available in the RST `guide_spatially_structured_networks`, there is no need for the PDF and no point in citing it.

Reply:
Yes, it does sound a bit cumbersome! I changed it to the phrase you suggested with a full stop after ``[1]_``. The link shows up as the title of the guide (``Guide to spatially structured networks``) and points to the RST version of the guide. The PDF has been removed.  

You can preview the documentation locally using Sphinx. Sure you already saw the instructions in #1439 :)

The figures are added in the reStructured text using ``.. figure::`` and the path. The images are located in ``doc/guides/spatial/user_manual_figures``. The code examples are embedded using ``.. literalinclude::`` and can be found in ``doc/guides/spatial/user_manual_scripts``.